webpack.config.js

```diff
@@ -5,6 +5,7 @@ const dist = path.resolve(__dirname, "dist");
 const WasmPackPlugin = require("@wasm-tool/wasm-pack-plugin");
 
 module.exports = {
+  watch: true,
```

Comment:
This is default in `webpack-dev-server` so I hope we will not need this

Reply:
Right, I thought so to, but when I logged the `compiler.options.watch` in https://github.com/wasm-tool/wasm-pack-plugin I got `undefined` if I didn't set it to true.

Reply:
I think it should run the other way around. And for sure webpack-dev-server adds this by default. I think I found the issue. I will raise a PR for `wasm-pack-plugin`

Reply:
Alright, great. Then we can close this PR once that's merged.